front_end/object_ui/CustomPreviewComponent.js

```diff
@@ -213,6 +213,37 @@ ObjectUI.CustomPreviewComponent = class {
     this.element = createElementWithClass('span', 'source-code');
     const shadowRoot = UI.createShadowRootWithCoreStyles(this.element, 'object_ui/customPreviewComponent.css');
     this.element.addEventListener('contextmenu', this._contextMenuEventFired.bind(this), false);
+
+    // DDT-specific operation to linkify an object's displayed runtime type
+    // Performed on all remote objects for which the following is resolvable:
+    // (Javascript) remoteObj["runtimeType"][Symbol(_type)]["[[FunctionLocation]]"]
+    // In Dart, this corresponds to an object's constructor's function location
+    let remoteFunction = "function remoteFunction() { " +
```

Comment:
Do we need to name this?

Reply:
After further testing, leaving it anonymous seems to work. Updated.